xds/type/matcher/v3/string.proto

```diff
@@ -52,6 +52,25 @@ message StringMatcher {
     //
     // * *abc* matches the value *xyz.abc.def*
     string contains = 7 [(validate.rules).string = {min_len: 1}];
+
+    // The input string must have the pattern specified here.
+    // A pattern is defined as a set of values between "/"s.
+    // Each value can be a string literal or wildcard (*).
+    //
+    // Empty pattern match is not allowed.
+    //
+    // Note: There is overlap between regex and pattern.
+    // Pattern serves as a better user experience through simplified matching syntax.
+    //
+    // Examples:
+    //
+    // abc/*/* matches the value abc/eng/us
+    // Note format of: abs/<wildcard-1>/<wildcard-2>
+    // {wildcard-1} == "eng" and {wildcard-2} == "us"
```

Comment:
I think that wildcard-1/2 makes this more confusing, as the matcher only matches the entries.

Reply:
changed